### PR TITLE
Added javascript exercises 1 and 2 for day 1

### PR DESCRIPTION
**Pull request by charlesnummer**
Added 2 javascript exercises and merged with main.

### CODE REVIEW

**Comment by charlesnummer** [comment]
JavascriptExercise1
JavascriptExercise2

**Comment by charlesnummer** [comment]
JavascriptExercise4Multiply
3.png)
JavascriptExercise5

**Comment by charlesnummer** [comment]
JavascriptExercise4Multiply

**Comment by charlesnummer** [comment]
JavascriptUdemy

File: charlesnummer/javascript/js2-exercise4.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                firstNumber = document.getElementById("myform").elements["1st Number"].value;
+                secondNumber = document.getElementById("myform").elements["2nd Number"].value;
+                final = firstNumber * secondNumber;
+                document.getElementById("final").textContent = final;
+            }
+            function divide() {
+                firstNumber = document.getElementById("myform").elements["1st Number"].value;
+                secondNumber = document.getElementById("myform").elements["2nd Number"].value;
+                final = firstNumber / secondNumber;
+                document.getElementById("final").textContent = final;
```
Comment (madhanramu) [inline charlesnummer/javascript/js2-exercise4.html:43]: good

File: charlesnummer/javascript/js2-exercise5.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Javascript Exercise 5</title>
+    </head>
+    <body>
+        <div>
+            <h1>Products</h1>
+            <ul>
+                <li>Product 1 <button onclick="add1()">Add to wishlist</button></li>
```
Comment (madhanramu) [inline charlesnummer/javascript/js2-exercise5.html:10]: use a single method add() for all the 3 products. On click of "Add to Wishlist". The element should get removed from "Products" and should get added to "Wishlist"

**Review by madhanramu — CHANGES_REQUESTED** [review]

**Pull request merged**